release.yml

```diff
@@ -54,10 +56,17 @@ steps:
       --configuration $(buildConfiguration)
       --output $(Build.ArtifactStagingDirectory)/nupkgs
       --no-build
-      -p:NuspecProperties="version=$(major).$(minor).$(Build.BuildNumber)"
+      -p:NuspecProperties="version=$(version)"
```

Comment:
Where do we define the `version` variable?

Reply:
It is defined in Line 20.

Reply:
Sorry for missing the folded snippets : )